crates/iota/src/iota_commands.rs

```diff
@@ -274,6 +274,8 @@ pub enum IotaCommand {
         #[clap(long, name = "iota|<full-url>")]
         #[arg(num_args(0..))]
         remote_migration_snapshots: Vec<SnapshotUrl>,
+        #[clap(long, help = "Specify the delegator address")]
+        delegator: Option<String>,
```

Comment:
I think `IotaAddress` can be used directly here.

Reply:
Currently, the delegator is supplied as stardust address. Are we planning to supply it as IotaAddress (0x..) for mainnet?

Reply:
Yes the delegator will come from the Destination Address of the Address Swap Map #4314.

Reply:
ok, done